### PR TITLE
Rolling reboot for Galera PC compatibility

### PR DESCRIPTION
**Pull request by maximumG**
When configuring a Galera cluster, we need to do a rolling reboot of all cluster members (instead of rebooting all node the same time) in order to preserve the Galera Primary Component up.

### CODE REVIEW

**Comment by gbaudot** [comment]
[changlog](https://fossies.org/linux/ansible/changelogs/CHANGELOG-v2.9.rst)
I had doubts yesterday... and checked today: throttle keyword came with version 2.9.0, then ansible_min_version in meta file should be changed accordingly.

**Comment by HanXHX** [comment]
Hi,

Can you update (remove ansible 2.8 reference) `.travis.yml` too please? It tests ansible version...

I had a look on travis... And I see another problem on CI (never seen before). I need to work on this :(

CI public page: https://travis-ci.org/github/HanXHX/ansible-mysql/builds/697146926

It seems it's related to: https://github.com/hashicorp/vagrant/issues/10022

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 env:
-  - PLATFORM='docker-buster-default-master'     ANSIBLE_VERSION='ansible>=2.8,<2.9'
-  - PLATFORM='docker-buster-upstream-master'    ANSIBLE_VERSION='ansible>=2.8,<2.9'
-  - PLATFORM='docker-buster-default-galera-1'   ANSIBLE_VERSION='ansible>=2.8,<2.9'
-  - PLATFORM='docker-buster-upstream-galera-1'  ANSIBLE_VERSION='ansible>=2.8,<2.9'
+  - PLATFORM='docker-buster-default-master'     ANSIBLE_VERSION='ansible>=2.9,<2.10'
```
Comment (HanXHX) [inline .travis.yml:2]: You don't need to update 2.8 -> 2.9, it's already done bellow.

Please remove all 2.8 related line :smile:

**Review by HanXHX — CHANGES_REQUESTED** [review]

**Comment by HanXHX** [comment]
I fixed Travis crashes int #41 

Can you rebase `master` in you branch please? I think it's all ok after that!

**Pull request merged**